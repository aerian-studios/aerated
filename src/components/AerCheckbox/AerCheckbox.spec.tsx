import React from "react";
import { describe, it, expect } from "vitest";
import { render, cleanup } from "@testing-library/react";
import { AerCheckbox } from "./index";

describe("AerCheckbox", () => {
    afterEach(() => {
        cleanup();
    })
    
    it("should render correctly", () => {
        const { container } = render(<AerCheckbox />);
        expect(container).toMatchSnapshot();
    });
});
