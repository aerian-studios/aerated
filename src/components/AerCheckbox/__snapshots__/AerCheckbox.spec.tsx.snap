// Vitest Snapshot v1

exports[`AerCheckbox > should render correctly given just label and value 1`] = `
<div>
  <div
    class="_wrapper_d61275"
  >
    <label
      class="_component_d61275"
    >
      <span
        class="_indicator_d61275"
      >
        <input
          class="_input_d61275"
          type="checkbox"
          value="sut"
        />
        <span
          aria-hidden="true"
          class="_imgWrapper_d61275 _checkBox_d61275"
        />
      </span>
      <span
        class="_label_d61275"
      >
        checkbox label
      </span>
    </label>
  </div>
</div>
`;
